rules/alerting.go

```diff
@@ -240,6 +245,7 @@ func (rule *AlertingRule) HTMLSnippet(pathPrefix string) template.HTML {
 	}
 	s += fmt.Sprintf("\n  SUMMARY %q", rule.summary)
 	s += fmt.Sprintf("\n  DESCRIPTION %q", rule.description)
+	s += fmt.Sprintf("\n  RUNBOOK <a href=%q>%s</a>", rule.runbook)
```

Comment:
As concluded, probably best to not assume it must be a link. Also, I'm not sure if `%q` will quote properly for HTML.